charters/DataBrowser/charter.md

```diff
@@ -0,0 +1,66 @@
+# Data Browser
+
+
+## Description
+The Data Browser (available via the Data Portal 'Explore' section) is an application that provides a faceted search capability allowing users to search and download subsets of cellular resolution data hosted by the Data Coordination Platform (DCP).
+
```

Comment:
Should this mention the goal of acting like a shopping cart for analysis applications? that isn't evident from this description?

Reply:
Added such a clause to the Description.